docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          step-name: 'Build my Unity project'
+```
+
+#### unity-license-var-name
+
+- _Default_: `UNITY_ENCODED_LICENSE`
+- _Required_: `False`
+- _Type_: `env_var_name`
+
+The name of the environment variable holding the license set in your
+[Context](/docs/circleci/activation#preparing-the-context). If you are building your project using
+the Unity Pro or Plus license, this is **not** required. See [Activation](/docs/circleci/activation)
+for more details.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+```
+
+#### unity-password-var-name
+
+- _Default_: `UNITY_PASSWORD`
+- _Required_: `False`
+- _Type_: `env_var_name`
+
+The name of the environment variable holding the password set in your
+[Context](/docs/circleci/activation#preparing-the-context). See
+[Activation](/docs/circleci/activation) for more details.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          unity-password-var-name: 'UNITY_PASSWORD'
+```
+
+#### unity-serial-var-name
+
+- _Default_: `UNITY_SERIAL`
+- _Required_: `False`
+- _Type_: `env_var_name`
+
+The name of the environment variable holding the serial set in your
+[Context](/docs/circleci/activation#preparing-the-context). If you are building your project using
+the Unity Personal license, this is **not** required. See [Activation](/docs/circleci/activation)
+for more details.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          unity-serial-var-name: 'UNITY_SERIAL'
+```
+
+#### unity-username-var-name
+
+- _Default_: `UNITY_USERNAME`
+- _Required_: `False`
+- _Type_: `env_var_name`
+
+The name of the environment variable holding the username set in your
+[Context](/docs/circleci/activation#preparing-the-context). See
+[Activation](/docs/circleci/activation) for more details.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          unity-username-var-name: 'UNITY_USERNAME'
+```
+
+### Examples
+
+In this sub-section, you will find examples of workflows using this job. However, notice that the
+separation used in the examples is for visualization's sake only. You can build for all target
+platforms in the same workflow.
+
+#### Personal License
+
+Below you will see how the build job can be used to build your project for several platforms using
+Unity's personal license. If you are unsure what this means, navigate to the
+[Activation](/docs/circleci/activation) section. All examples are based on CircleCI's
+[Demo Project](https://github.com/CircleCI-Public/Unity2D-Demo-Game-CI-CD).
```

Comment:
```suggestion
This examples shows how to build your project for several platforms using a Personal License. 
```

Reply:
I'm just replacing `examples` with `example`. And done ✔️